includes/rules.php

```diff
@@ -74,6 +74,14 @@
 			'<code>&lt;h2&gt;</code>'
 		),
 	),
+	array(
+		'title'     => esc_html__( 'Empty Paragraph Tags', 'accessibility-checker' ),
+		'info_url'  => 'https://a11ychecker.com/help###',
+		'slug'      => 'paragraph_should_not_be_empty',
+		'rule_type' => 'error',
+		'summary'   => esc_html__( 'Paragraph tags that are empty get announced to screen readers with empty content creating unnecessary noise. Paragraphs should never be used for layout purposes.', 'accessibility-checker' ),
```

Comment:
An Empty Paragraph Tag error means there is a paragraph tag present that does not contain content. These are announced by screen readers and create unnecessary noise for blind users. To fix this error, remove the empty paragraphs from the page. If you need to add spacing between sections, this should be done with padding, margins, or a spacer block.